docs/variants/clevo_nv41/releases.md

```diff
@@ -8,6 +8,35 @@ For details about our release process please read
 
 [Subscribe to Clevo NV41MZ Dasharo Release Newsletter](https://newsletter.3mdeb.com/subscription/T61MyO2sP){ .md-button .md-button--primary .center }
 
+## v1.1.0 - 2022-03-23
+
+### Added
+
+- Add Dasharo Tools Suite network boot integration
+- Add a persistent bootlogo implementation
+
+### Known issues
+
+- [UCM-UCSI ACPI device displays an error in Windows Device Manager](https://gitlab.com/novacustom/dasharo-compatibility/-/issues/19)
+- [USB4 Root Device Router device displays an error in Windows 11 Device Manager](https://gitlab.com/novacustom/dasharo-compatibility/-/issues/18)
+- [Windows 10 SD card reader driver needs manual installation on NV41MB](https://gitlab.com/novacustom/dasharo-compatibility/-/issues/16)
+- [Low level interfering crackling/popping of the speakers while playing no sound](https://gitlab.com/novacustom/dasharo-compatibility/-/issues/13)
+- [Laptop cannot output video via the Tunderbolt 4 USB Type-C port](https://gitlab.com/novacustom/dasharo-compatibility/-/issues/7)
+
+
+### Binaries
+
+[clevo_nv41mz_v1.1.0.rom](https://3mdeb.com/open-source-firmware/Dasharo/clevo_nv41mz/clevo_nv41mz_v1.1.0.rom){ .md-button }
+[sha256](https://3mdeb.com/open-source-firmware/Dasharo/clevo_nv41mz/clevo_nv41mz_v1.1.0.rom.sha256){ .md-button }
+[sha256.sig](https://3mdeb.com/open-source-firmware/Dasharo/clevo_nv41mz/clevo_nv41mz_v1.1.0.rom.sha256.sig){ .md-button }
```

Comment:
@mkopec doesn't pre-commit complain on those long lines? I noted that if you have bullet list then pre-commit don't care about line length otherwise there are tricks need to avoid problems and complains with this lines. Of course because this is button it probably cannot be in bullet list.

This is how I [resolved that in fix_branch](https://github.com/Dasharo/docs/pull/110/commits/98d4a7f93fbb1056344eef8789ef0802a0bfd279). I'm not sure if this is acceptable strategy so please take it with grain of salt. @macpijan please let me know also what is your view on that.

Reply:
@macpijan I also have similar feeling that it may be hard to maintain, alternatively we can add some short labels, we just need some pattern. I see no need for re-enumeration after adding new release, numbers are just labels which can increase forever.

I don't know if maintenance of that would be expensive, we doing that just once and we should never change that. What is the reason to change links/labels for some old releases?

Proposed fix, not talking about tags, will not work for following reasons:

1. [This](https://github.com/Dasharo/docs/commit/f0ac124e7cfbff5e9fdb06edb7bae0dfa791f913#diff-b0f1b7a0ce7eb3de7caeb1c99af3af7c4c4944b7c4475e94d295f2522500efd6R60) is already done in many palaces but link still is longer then 80 characters.
2. [This](https://github.com/Dasharo/docs/commit/f0ac124e7cfbff5e9fdb06edb7bae0dfa791f913#diff-b0f1b7a0ce7eb3de7caeb1c99af3af7c4c4944b7c4475e94d295f2522500efd6R9) will not work for Talos II because name is too long. I proposed change of button label which always would be shorter [here](https://github.com/Dasharo/docs/pull/110/commits/98d4a7f93fbb1056344eef8789ef0802a0bfd279#diff-342c0ec9d784cffe1926a64dfd8e7690953a4fdc1a618216eb3d9b051409f24fR9)

From what I understand tags may be good solution, but reasonable labeling could also work. If we talking about labels there is standard for that in CommonMark which is used by pre-commit [here](https://spec.commonmark.org/0.30/#link-reference-definitions), but if we agree about using tags then let's do it.

Reply:
@macpijan As I mentioned in https://github.com/Dasharo/docs/pull/110#issuecomment-1076510267, the solution proposed in https://github.com/Dasharo/docs/pull/110#issuecomment-1074884316 is not perfect

Reply:
Maybe let's try with the tags then. It would be nice if they can follow some rule (e.g. component_version) so they can be automatically generated.

Reply:
@macpijan The tags can be named and placed anywhere in the doc it seems, so i.e. this will render correctly:

```
[clevo_nv41mz_v1.1.0.rom][v1.1.0_rom]{ .md-button }
[sha256][v1.1.0_hash]{ .md-button }
[sha256.sig][v1.1.0_sig]{ .md-button }

[v1.1.0_rom]: https://3mdeb.com/open-source-firmware/Dasharo/clevo_nv41mz/clevo_nv41mz_v1.1.0.rom
[v1.1.0_hash]: https://3mdeb.com/open-source-firmware/Dasharo/clevo_nv41mz/clevo_nv41mz_v1.1.0.rom.sha256
[v1.1.0_sig]: https://3mdeb.com/open-source-firmware/Dasharo/clevo_nv41mz/clevo_nv41mz_v1.1.0.rom.sha256.sig
```

I'll update the entire release page and check if this satisfies the linter

Reply:
@mkopec @macpijan I like this solution. @mkopec would you mind to fix that in #110 ? With new changes I will follow that rule, maybe we should also write that somewhere or enforce in pre-commit?

Reply:
@pietrushnic Sure, I'll fix it in #110 too

Reply:
Applied to nv41 in #110 too

Reply:
@pietrushnic Applied to Talos and KGPE-D16 as well